consensus-client-it/src/test/scala/units/TestNetworkClient.scala

```diff
@@ -0,0 +1,53 @@
+package units
+
+import com.wavesplatform.network.message.Message
+import com.wavesplatform.network.{Handshake, LegacyFrameCodecL1, PeerDatabase, RawBytes}
+import io.netty.bootstrap.Bootstrap
+import com.wavesplatform.account.Address
+import io.netty.channel.nio.NioIoHandler
+import io.netty.channel.socket.SocketChannel
+import io.netty.channel.socket.nio.NioSocketChannel
+import io.netty.channel.*
+import io.netty.handler.codec.LengthFieldPrepender
+import units.network.BlockSpec
+
+import scala.concurrent.duration.*
+
+object TestNetworkClient {
+  private val applicationVersion = (1, 5, 7)
+  private val nodeName           = "test-client"
+
+  def send(address: String, port: Int, chainContractAddress: Address, block: NetworkL2Block): Unit = {
```

Comment:
What I see:
1. `address` is always "127.0.0.1"
2. `port` is always `node.networkPort`

It won't work, if you provide other values. How's about:
```suggestion
  def send(node: WavesNodeContainer, chainContractAddress: Address, block: NetworkL2Block): Unit = {
```
?